lib/config/sources/yaml_source.rb

```diff
@@ -9,7 +9,7 @@ class YAMLSource
 
       def initialize(path, evaluate_erb: Config.evaluate_erb_in_yaml)
         @path = path.to_s
-        @evaluate_erb = !!evaluate_erb
+        @evaluate_erb = evaluate_erb
```

Comment:
Not sure about this one...it's kind of nice having a proper boolean, since it is accessible via the attr_reader.